cmd/jujud/agent/machine/manifolds.go

```diff
@@ -782,6 +783,14 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 			NewWorker:  controlsocket.NewWorker,
 			SocketName: paths.ControlSocket(paths.OSUnixLike),
 		})),
+
+		sshServerName: ifController(sshserver.Manifold(sshserver.ManifoldConfig{
```

Comment:
may i suggest adding this in the last step of the process, once we have everything wired up and fully implemented?

Reply:
I can't Q/A without it (i.e., bootstrap and literally hit the server)